be/src/vec/functions/function_string.h

```diff
@@ -356,6 +357,182 @@ class FunctionStrcmp : public IFunction {
     }
 };
 
+class FunctionAutoPartitionName : public IFunction {
+public:
+    static constexpr auto name = "auto_partition_name";
+    static FunctionPtr create() { return std::make_shared<FunctionAutoPartitionName>(); }
+    String get_name() const override { return name; }
+    size_t get_number_of_arguments() const override { return 0; }
```

Comment:
remove the useless code

Reply:
"get_number_of_arguments" is pure virtual functions which should be implemented